.github/workflows/ci-pkg-install.yml

```diff
@@ -12,6 +12,7 @@ on:
       - "setup.py"
       - "src/**"
       - "requirements/**"
+      - "!requirements/*/docs.txt"
```

Comment:
Shouldn't this have 2 `*`?

Reply:
so far I know `**` is for any depth and `*` is just one folder

Reply:
But in the line above you use `**`

Reply:
i could be wrong, but... 
- `**` would cover files in requirements and all subfolders
- `*` would cover only files directly in requirements but no subfolder

Reply:
I believe that this is the same pattern: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/util/pattern/PathPattern.html